<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainw</class>
 <widget class="QWidget" name="mainw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainw</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_eraseFlash">
     <property name="text">
      <string>Erase Flash</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_detectedDevices"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_deviceToFlash">
     <property name="text">
      <string>Device to flash</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_detectDevices">
     <property name="text">
      <string>Detect devices...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_firmwareFile">
     <property name="text">
      <string>Firmware File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_status">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_firmwareFile"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_flashFirmware">
     <property name="text">
      <string>Flash Firmware</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_selectFile">
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTextEdit" name="log_text">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
